api/v1beta1/kubevirtmachinetemplate_types.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// KubevirtMachineTemplateSpec defines the desired state of KubevirtMachineTemplate.
+type KubevirtMachineTemplateSpec struct {
```

Comment:
After giving this some more thought, I'm unsure if we should be incrementing the capk object's to beta, but instead ensure that we work properly with the capi v1beta1 apis.

Our capk apis are still alpha, and will certainly change (see #11 ) so moving to beta is probably premature. That hopefully doesn't inhibit us adopting the capi v1beta1 api though. It seems like these two things should be orthogonal to one another.